### DISCUSSION TITLE
How to migrate 'sub-collection' of fireStore to supabase?

### DISCUSSION THREAD

**Discussion by hayatshin** [supabase/supabase#19910] [Questions]
I tried to migrate fireStore db to supabase.

The process for this is documented well in official documents:
https://supabase.com/docs/guides/resources/migrating-to-supabase/firestore-data

But when I run node collections.js to list all collections of fireStore, it shows all collections only.
fireStore has concept subCollection and I want to make it as separate Table in postgreSQL.

Even if that's not the case, I need to fetch data from sub-collection from fireStore.

Is it impossible now?

**Comment by JeffLoucks** [comment]
Seems like this is a common problem. I found this blog post to really helpful: https://dev.to/ja/how-i-migrated-from-firebase-to-supabase-3n56

It uses the `node-firestore-import-export` npm package to export your entire firestore db into a json file, and then they use a script (which you can copy from that blog post) to split each collection and subcollection into separate json files.
